common/azure-common/src/main/java/com/azure/common/exception/ServiceResponseException.java

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.common.exception;
+
+/**
+ * The request was sent, but the client failed to understand the response.
+ * These errors may not be safe to retry
```

Comment:
The 'These errors' sentence needs to be fleshed out - what should the user do in this situation?